src/cpu/pred/BranchPredictor.py

```
@@ -1044,15 +1044,15 @@ class BTBTAGE(TimedBaseBTBPredictor):
     enableSC = Param.Bool(False, "Enable SC or not")    # TODO: BTBTAGE doesn't support SC
     updateOnRead = Param.Bool(True, "Enable update on read, no need to save tage meta in FTQ")
     numPredictors = Param.Unsigned(8, "Number of TAGE predictors")
-    tableSizes = VectorParam.Unsigned([2048, 2048, 2048, 2048, 2048, 2048, 2048,2048],"the TAGE T0~Tn length")
+    tableSizes = VectorParam.Unsigned([8192] * 8, "the TAGE T0~Tn length")
```

Ruff (0.15.6)

[error] `VectorParam` may be undefined, or defined from star imports

(F405)